docs/ref/forms/api.txt

```diff
@@ -260,9 +262,30 @@ precedence::
 
 .. method:: Form.get_initial_for_field(field, field_name)
 
-Use :meth:`~Form.get_initial_for_field()` to retrieve initial data for a form
-field. It retrieves data from :attr:`Form.initial` and :attr:`Field.initial`,
-in that order, and evaluates any callable initial values.
+Returns the initial data for a form field. The data is retrieved from
+:attr:`Form.initial` if present, otherwise :attr:`Field.initial`, and
+evaluated if a callable.
```

Comment:
Maybe...

```suggestion
Returns the initial data for a form field. It retrieves data from
:attr:`Form.initial` if present, otherwise trying :attr:`Field.initial`. 
Callable values are evaluated.
```